File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/EventDetails.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class EventDetails
{
	[JsonProperty("timestamp")]
	public double TimeStamp { get; set; }

	[JsonProperty("items")]
	public List<EventItem> Items { get; set; }

	[JsonProperty("http_performance_data")]
	public HttpPerformanceData HttpPerformanceData { get; set; }

	[JsonProperty("http_request_details")]
	public HttpRequestDetails HttpRequestDetails { get; set; }

	[JsonProperty("timing")]
	public TimingDetails TimingDetails { get; set; }

	[JsonProperty("share")]
	public ShareDetails ShareDetails { get; set; }

	[JsonProperty("launch")]
	public LaunchDetails LaunchDetails { get; set; }

	[JsonProperty("alert")]
	public AlertDetails AlertDetails { get; set; }
}

File: Android/Vine-5.50.1/Decompilation/src/java/co/vine/android/scribe/model/EventDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;
import com.mobileapptracker.MATEvent;
import java.util.List;

@JsonObject
/* loaded from: classes.dex */
public class EventDetails {

    @JsonField(name = {"alert"})
    public AlertDetails alert;

    @JsonField(name = {"http_performance_data"})
    public HTTPPerformanceData httpPerformanceData;

    @JsonField(name = {"http_request_details"})
    public HTTPRequestDetails httpRequestDetails;

    @JsonField(name = {"items"})
    public List<Item> items;

    @JsonField(name = {"launch"})
    public LaunchDetails launch;

    @JsonField(name = {"playback_summary"})
    public PlaybackSummaryDetails playbackSummary;

    @JsonField(name = {MATEvent.SHARE})
    public ShareDetails share;

    @JsonField(name = {"timestamp"})
    public Double timestamp;

    @JsonField(name = {"timing"})
    public TimingDetails timing;

    @JsonField(name = {"video_import_details"})
    public VideoImportDetails videoImportDetails;
}
